### PR TITLE
log: add logs to dkg

### PR DESCRIPTION
**Pull request by xenowits**
Add more debug/info logs to dkg ceremony

category: refactor
ticket: #587

### CODE REVIEW

**Review by leolara — APPROVED** [review]

File: dkg/dkg.go [hunk 1]
```diff
@@ -142,29 +142,38 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return errors.New("unsupported dkg algorithm")
 	}
 
-	log.Info(ctx, "Successfully completed DKG ceremony, writing output")
-
+	log.Info(ctx, "Writing keyshares to disk")
```
Comment (corverroos) [inline dkg/dkg.go:145]: why is this info and the rest debug?

Comment (xenowits) [reply dkg/dkg.go:145]: good catch, it's one of the debug guys disguising as info. i'll change it

File: dkg/dkg.go [hunk 2]
```diff
@@ -142,29 +142,38 @@ func Run(ctx context.Context, conf Config) (err error) {
...
 		return err
 	}
 
+	log.Debug(ctx, "Writing lock file to disk")
 	if err = writeLock(conf.DataDir, lock); err != nil {
 		return err
 	}
 
 	// Sign, exchange and aggregate Deposit Data signatures
+	log.Debug(ctx, "Aggregating deposit data signatures")
```
Comment (dB2510) [inline dkg/dkg.go:163]:
```suggestion
	log.Debug(ctx, "Aggregating deposit data signatures...")
```

Comment (dB2510) [reply dkg/dkg.go:163]: likewise everything above maybe

File: dkg/dkg.go [hunk 3]
```diff
@@ -142,29 +142,38 @@ func Run(ctx context.Context, conf Config) (err error) {
...
 	aggSigDepositData, err := signAndAggDepositData(ctx, ex, shares, def.WithdrawalAddress, network, nodeIdx)
 	if err != nil {
 		return err
 	}
 
-	return writeDepositData(aggSigDepositData, def.WithdrawalAddress, network, conf.DataDir)
+	log.Debug(ctx, "Writing deposit data file to disk")
```
[the remark this answers is not in the document]
Comment (xenowits) [reply dkg/dkg.go:169]: i have tried to avoid this, not sure to add it

[the remark this answers is not in the document]
Comment (corverroos) [reply dkg/dkg.go:169]: I'm also not sure when to use `...` and when not. But we should probably be consistent. Easiest is not not include it.

[the remark this answers is not in the document]
Comment (dB2510) [reply dkg/dkg.go:169]: yeah, i thought `...` means something is happening

[the remark this answers is not in the document]
Comment (corverroos) [reply dkg/dkg.go:169]: yeah sure. But should it always be used for present tense logs? 

Other option is to write the log after the action in the past tense: `Saved deposit data file to disk`

[the remark this answers is not in the document]
Comment (dB2510) [reply dkg/dkg.go:169]: Like we are doing in create cluster

[the remark this answers is not in the document]
Comment (corverroos) [reply dkg/dkg.go:169]: Personally, I would probably say that `...` should only be used before one or two long running/slow actions. If actions are fast, or if there is a lot of them, I would not use `...`. But it doesn't really matter I think. Up to you guys.

[the remark this answers is not in the document]
Comment (xenowits) [reply dkg/dkg.go:169]: yeah there are 2 options here though:
 i). log before the action takes place
ii). log after the action takes place

**Pull request merged**